### PR TITLE
docs: minor suggetions to 3.0.0 releases

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This is a practice PR. This PR also contains minor suggestions for how to improve the release notes.

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/940#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Welcome @karl-cardenas-coding! It looks like this is your first PR to spectrocloud/librarium 🎉

**Comment by spectro-prow** [comment]
Hi @karl-cardenas-coding. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by karl-cardenas-coding** [comment]
Hi @resmaa, Anton asked to me take a peek at the release notes. I added some suggestions that I am happy to chat through, so don't feel obligated to accept them. I'm still trying to get a feel for how things work here 😄

File: content/docs/00-release-notes.md
Comment (resmaa) [inline content/docs/00-release-notes.md:24]:
```suggestion
* A new set of capabilities that improve the [developer experience](/devx) are introduced in this release:
```

File: content/docs/00-release-notes.md
Comment (resmaa) [inline content/docs/00-release-notes.md:36]:
```suggestion
* Palette enterprise mode production clusters can be backed up to [Azure Blob storage](/clusters/cluster-management/backup-restore#forazureblobbackup) for convenient restoration.
```

**Review by resmaa — CHANGES_REQUESTED** [review]
made minor changes
let's remove json file

**Comment by karl-cardenas-coding** [comment]
Closing PR due to accidentally including package lock

**Pull request closed without merging**
